packages/dialtone-css/lib/build/less/components/badge.less

```diff
@@ -135,15 +138,25 @@
     height: var(--dt-size-400);
     background-color: var(--badge-decorative-color);
     border-radius: var(--dt-size-200);
+    margin-inline-start: var(--dt-space-200);
   }
 
   &__label {
     display: flex;
     align-items: center;
+    padding-inline: var(--badge-label-padding-x);
   }
 
   &__icon-left,
   &__icon-right {
     display: flex;
   }
+
+  &__icon-left {
+    padding-inline-start: var(--dt-space-100);
+  }
+
+  &__icon-right {
+    padding-inline-end: var(--dt-space-100);
+  }
```

Comment:
You'll note that I'm starting to use logical CSS properties instead, e.g. `padding-inline-start` instead of `padding-left`, `mid-inline-size` instead of `min-width`, etc.